docs/source/conf.py

```diff
@@ -149,6 +150,21 @@
 napoleon_use_ivar = True
 napoleon_include_special_with_doc = True
 
+# -- Copybutton settings --------------------------------------------------
+
+# Only copy lines starting with the input prompts,
+# valid prompt styles: [
+#     Python Repl + continuation (e.g., '>>> ', '... '),
+#     Bash (e.g., '$ '),
+#     ipython and qtconsole + continuation (e.g., 'In [29]: ', '  ...: '),
+#     jupyter-console + continuation (e.g., 'In [29]: ', '     ...: ')
+# ]
+copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
```

Comment:
How about mentioning the regexp is taken from the official doc https://sphinx-copybutton.readthedocs.io/en/latest/#using-regexp-prompt-identifiers to be ready for possible updates?

Reply:
Good point @toslunar! Done ✅